ui_framework/src/components/pop_menu/_pop_menu.scss

```diff
@@ -0,0 +1,92 @@
+// Pop menu is an animated popover relatively positioned to a button / action.
+// By default it positions in the middle, but can be anchored left and right.
+
+@include component('kuiPopMenu') {
+  position: relative;
+
+  // Open state happens on the wrapper and applies to the body.
+  @include state('isOpen') {
+    .kuiPopMenu__body {
```

Comment:
There's a limitation to the Bemify script with nested children. I've run into something similar and will tweak their mixin, then clean this up in a separate PR.